src/bundle/Resources/views/themes/admin/ui/component/tab/tabs_header.html.twig

```diff
@@ -29,17 +29,17 @@
             </ul>
             {% if include_tab_more|default(false) %}
                 {{ include('@ezdesign/ui/component/popup_menu/popup_menu.html.twig', {
-                    'class': 'ibexa-tabs__popup-menu ibexa-tabs__popup-menu--hidden',
+                    'class': 'ibexa-tabs__popup-menu',
                     'items': [],
                 }) }}
             {% endif %}
             {% if not hide_toggler|default(false) %}
                 <div class="ibexa-tabs__toggler">
                     <span class="ibexa-tabs__toggler-show">
-                        {{ 'ui.component.tab.tabs.show'|trans|desc('Show') }}
+                        {{ 'ui.component.tab.tabs_toggler.show'|trans|desc('Show') }}
```

Comment:
Shouldn't this be extracted or such key exists?

Reply:
I haven't noticed this change. Probably, by a mistake I have forgot to commit it in previous PR, because in *xliff* it has correct key: https://github.com/ezsystems/ezplatform-admin-ui/pull/1780/files#diff-fc54d07c9121de0ecf10e28736fa4c69f6e811c8f35b03d18f0a11ba1ef6a353R457